### ISSUE TITLE
Home Page - Can't Use Keyboard Arrow Keys to Navigate Drop Down List

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#1]
While testing with the keyboard using DHS Trusted Tester process, I found a person could not navigate the drop down the list after opening the list with down arrow instead of tabbing into the field. After opening the drop down list by activating the down arrow was not able to move around the list with arrow keys.

Section 508 - 21(a)
WCAG 2.0 - 1.1 Keyboard Access

**Comment by adamsfallen** [comment]
This is true - the dropdown was added as a visual cue and it doesn't support tabbing into the dropdown. However, the list should work with a keyboard if you tab into the field itself as if it was a normal dropdown.

**Comment by jfc3-dol** [comment]
Why not make this a drop down list (SELECT) if you are only going to allow certain options and people can't enter there own. By having it as what looks like a fill in the blank field I would expect as a user to be able to enter my own type of user/person.

**Comment by adamsfallen** [comment]
The user can enter his or her own job title and that information is captured in our analytics to help inform future iterations, although you're right currently the site itself redirects to a generic page.
